metagpt/actions/ga_action_base.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# author: didi
+# Date:9.25
+
+import openai
+from metagpt.llm import DEFAULT_LLM
+# 直接调用Prompt生成
+# ga的prompt构建格式和metagpt完全不同。没有办法融合。
+
+
+# 特殊指令加入Prompt生成
+
+
```

Comment:
action here should inherit from `Action` like `https://github.com/geekan/MetaGPT/blob/ga_game/examples/st_game/actions/summarize_conv.py`